pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2527,7 +2488,6 @@ self: super: {
   system-fileio = doJailbreak (dontCheck super.system-fileio);
 
   # Bounds too strict on base and ghc-prim: https://github.com/tibbe/ekg-core/pull/43 (merged); waiting on hackage release
-  ekg-core = assert super.ekg-core.version == "0.1.1.7"; doJailbreak super.ekg-core;
   hasura-ekg-core = doJailbreak super.hasura-ekg-core;
```

Comment:
`ekg-core` sits at 0.1.1.8 so this assert was failing, but `hasura-ekg-core` is a `non-hackage-packages` entry for the same lib at 0.1.1.7.
Not sure why it exists. Maybe it can be removed now?